src/zsh_completion/_firejail.in

```diff
@@ -105,6 +105,7 @@ _firejail_args=(
     '--keep-config-pulse[disable automatic ~/.config/pulse init]'
     '--keep-dev-shm[/dev/shm directory is untouched (even with --private-dev)]'
     '--keep-fd[inherit open file descriptors to sandbox]'
+    '--keep-xattr[copy specified extended attributes to sandbox]'
```

Comment:
```suggestion
    '*--keep-xattr=-[copy specified extended attributes to sandbox]'
```

- You can use `--keep-xattr` multiple times, right? => `*--`
- It takes a value. => `=-`

Reply:
```suggestion
    '*--keep-xattr=-[copy the specified extended attributes to the sandbox]'
```

Above changes + grammar improvements.